src/OpenColorIO/Platform.h

```diff
@@ -124,43 +124,43 @@ OCIO_NAMESPACE_ENTER
 //       In the meantime, hardcode to x86
 #define OCIO_LITTLE_ENDIAN 1  // This is correct on x86
 
-    /*
-     * Mutex classes
-     */
+/*
+ * Mutex classes
+ */
 
 #ifdef WINDOWS
 
-    class _Mutex {
-    public:
+class _Mutex {
+public:
 	_Mutex()       { _mutex = CreateMutex(NULL, FALSE, NULL); }
 	~_Mutex()      { CloseHandle(_mutex); }
 	void lock()   { WaitForSingleObject(_mutex, INFINITE); }
 	void unlock() { ReleaseMutex(_mutex); }
     private:
```

Comment:
Since this PR is targeting indentation. I would tend to maintain consistent access specifier indentation. `private:` could be unindented to match `public:`.